decidim_app-design/app/views/public/partials/_comments.html.erb

```diff
@@ -17,6 +17,9 @@
           </div>
           <div class="author-data__extra">
             <%= partial "flag_empty" %>
+            <a href="comment-view#single-comment" title="Enlace permamente">
```

Comment:
The link doesn't work in some cases. I think is due to the lack of a comment view detail of other entities like initiatives.
Would It be better to force the link to proposals/comment-view#single-comment?